src/s390_ecc.c

```diff
@@ -1743,16 +1744,26 @@ struct {				\
 #ifdef ICA_FIPS
 			if (fips & ICA_FIPS_MODE) {
 				fc |= 0x80; /* deterministic signature */
-				RAND_bytes(param.P256.rand + off, sizeof(param.P256.rand) - off);
+				do {
+					/* A cc=2 occurs if the random number is not invertible.
+					 * Loop in this case. Same for p384 and p521 below. */
```

Comment:
Should we mention `cc` here? The value is only visible in the inline-assembly function and is never returned to the caller.

Reply:
ok, yes, I will change the comment.